src/main/java/at/hannibal2/skyhanni/config/features/misc/MiscConfig.java

```diff
@@ -93,6 +93,11 @@ public class MiscConfig {
     @Accordion
     public TrackerConfig tracker = new TrackerConfig();
 
+    @Expose
+    @ConfigOption(name = "Leave/Join Messages", desc = "")
```

Comment:
maybe include here in the name and description that it is about players on your island. otherwise its confusing with party members, friends, guild members, etc

Reply:
Like `Island Leave/Join Messages`?